api/src/main/java/org/openmrs/ui/framework/FormatterImpl.java

```diff
@@ -113,6 +114,10 @@ private boolean wholeNumber(Number n) {
 
     private String format(Date d, Locale locale) {
         DateFormat df;
+        if(Boolean.parseBoolean(

```

Comment:
How about using `BooleanUtils.toBoolean()` to accept `on`, `yes`, `t`, `y` as true values?

Reply:
It makes a lot of sense, I changed it to use `BooleanUtils.toBoolean()` as you can see [here ](https://github.com/openmrs/openmrs-module-uiframework/pull/65/files#diff-920407a71f4d88e52f65862ff99257e9eae4b910d1f050f158937182bd5b3c5eR118) and [here](https://github.com/openmrs/openmrs-module-uiframework/pull/65/files#diff-6e177127ee5b17a0d0a088acefa4c4d980d03a506959223081b579f153e8382fR600).